web/src/utils/http.ts

```
@@ -72,14 +74,14 @@ http.interceptors.response.use(
     else {
       switch (response?.data?.code) {
         case ResultCode.UNAUTHORIZED: {
-          const logout = useDebounceFn(
-            async () => {
+          const logout = async () => {
+            if (isLogout === false) {
+              isLogout = true
+              setTimeout(() => isLogout = false, 5000)
               Message.error('登录状态已过期，需要重新登录', { zIndex: 9999 })
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)